ttnctl/cmd/devices_set.go

```diff
@@ -204,5 +213,6 @@ func init() {
 	devicesSetCmd.Flags().Int32("altitude", 0, "Set altitude")
 
 	devicesSetCmd.Flags().String("description", "", "Set Description")
-	devicesSetCmd.Flags().StringArray("custom-attr", []string{}, "Set a custom attribute")
+	devicesSetCmd.Flags().StringArray("attr-add", []string{}, "Add an additional attribute (ex:--attr-add <name>:<value>)")
```

Comment:
What is an _additional_ attribute other than just attribute? I don't think that ID is an attribute of a device; I think that all custom things are attributes